outgoing/InflightAssetsMessageProvider.sys.mjs

```diff
@@ -0,0 +1,306 @@
+      },
+      {
+        "id": "CFR_FULL_VIDEO_SUPPORT_EN",
+        "groups": [
+          "cfr"
+        ],
+        "targeting": "firefoxVersion < 88 && firefoxVersion != 78 && localeLanguageCode in ['en', 'fr', 'de', 'ru', 'zh', 'es', 'it', 'pl']",
+        "template": "cfr_doorhanger",
+        "content": {
+          "skip_address_bar_notifier": true,
+          "persistent_doorhanger": true,
+          "anchor_id": "PanelUI-menu-button",
+          "layout": "icon_and_message",
+          "text": {
+            "string_id": "cfr-doorhanger-video-support-body"
+          },
+          "buttons": {
+            "secondary": [
+              {
+                "label": {
+                  "string_id": "cfr-doorhanger-extension-cancel-button"
+                },
+                "action": {
+                  "type": "CANCEL"
+                }
+              }
+            ],
+            "primary": {
+              "label": {
+                "string_id": "cfr-doorhanger-video-support-primary-button"
+              },
+              "action": {
+                "type": "OPEN_URL",
+                "data": {
+                  "args": "https://support.mozilla.org/kb/update-firefox-latest-release",
+                  "where": "tabshifted"
+                }
+              }
+            }
+          },
+          "bucket_id": "CFR_FULL_VIDEO_SUPPORT_EN",
+          "heading_text": {
+            "string_id": "cfr-doorhanger-video-support-header"
+          },
+          "info_icon": {
+            "label": {
+              "string_id": "cfr-doorhanger-extension-sumo-link"
+            },
+            "sumo_path": "extensionrecommendations"
+          },
+          "notification_text": "Message from Firefox",
+          "category": "cfrFeatures"
+        },
+        "frequency": {
+          "lifetime": 3
+        },
+        "trigger": {
+          "id": "openURL",
+          "patterns": [
+            "https://*/Amazon-Video/*",
+            "https://*/Prime-Video/*"
+          ],
+          "params": [
+            "www.hulu.com",
+            "hulu.com",
+            "www.netflix.com",
+            "netflix.com",
+            "www.disneyplus.com",
+            "disneyplus.com",
+            "www.hbomax.com",
+            "hbomax.com",
+            "www.sho.com",
+            "sho.com",
+            "www.directv.com",
+            "directv.com",
+            "www.starzplay.com",
+            "starzplay.com",
+            "www.sling.com",
+            "sling.com",
+            "www.facebook.com",
+            "facebook.com"
+          ]
+        }
+      },
+      {
+        "id": "WNP_MOMENTS_12",
```

Comment:
I don't think these moments message are live any more, will open a separate pull request moving these moments messages to archive